client/src/shared/layouts/Layout-learning/SideBar.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link';
+import React from 'react';
+import MenuLogo from '../../icons/menu';
+import PreviousIcon from '../../icons/previous';
+
```

Comment:
- change file name to `sidebar`
- change file directory
client/src/shared/layouts/sidebar.tsx